src/index.ts

```diff
@@ -171,4 +195,89 @@ export class EmbeddingIndex {
       console.log(`Item ${idx + 1}:`, obj);
     });
   }
+
+  async saveIndexToDB(DBname: string = 'defaultDB', objectStoreName: string = 'DefaultStore'): Promise<void> {
```

Comment:
As I mentioned in the Readme, `saveIndex("indexedDB", { DBName: "clientVectorDB", objectStoreName:"ClientEmbeddings"})` is a nicer approach to this